recipes/install_monitoring.rb

```diff
@@ -51,6 +51,10 @@
 end
 
 if node['abiquo']['monitoring']['db']['install']
+    package "MariaDB-client" do
+    	action :install
+    end
```

Comment:
Add the corresponding assertions in [these expect tests](https://github.com/abiquo/abiquo-cookbook/blob/master/spec/install_monitoring_spec.rb#L80-L94).
Also add the verification that the package is actually installed in [the kitchen test](https://github.com/abiquo/abiquo-cookbook/blob/master/test/integration/monitoring/serverspec/monitoring_packages_spec.rb).